### PR TITLE
clean up packages and remove viper

### PR DESCRIPTION
**Pull request by davidgamero**
# Description
implementing the changes from #137 that @imiller31 made accounting for drift
- move several files into the folders they logically belong in
- remove unused extra web files
- remove remnants of viper usage in favor of full migration to yaml library

Fixes # (issue)
Feature # (details)

## Type of change

- [x] Refactor

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration. Is it a  breaking change which will impact consuming tool(s).

- [x] Existing integration tests
- [x] Existing unit tests  

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: cmd/create.go [hunk 1]
```diff
@@ -10,7 +9,7 @@ import (
 	"github.com/manifoldco/promptui"
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
-	"github.com/spf13/viper"
+	"sigs.k8s.io/kustomize/kyaml/yaml"
```
Comment (imiller31) [inline cmd/create.go:12]: Should we be using `kyaml` or yaml v3 here?

**Review by Tatsinnit — DISMISSED** [review]
💡👍 Thank you for adding me to the review, Just eye-balled and looks good to me.

Please Note: have we checked if this impact any behaviour specially any impact on `az-cli` or any other consuming tool, if so we can parallel change things in consuming tools please, also bunch of tests failing. Thanks.

**Comment by davidgamero** [comment]
> 

this change should be purely internal to organize the packages, and does not impact the usage/outputs at all, so no tests should have to be updated and azcli should retain all functionality

**Review by Tatsinnit — APPROVED** [review]

**Comment by imiller31** [comment]
LGTM

**Pull request merged**